### PR TITLE
CiderCollective.Cider.Canary version upcoming-411eeac

### PR DESCRIPTION
**Pull request by yazninja**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43731)

### CODE REVIEW

File: manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Canary
+PackageVersion: upcoming-411eeac
```
Comment (OfficialEsco) [inline manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Canary
+PackageVersion: upcoming-411eeac
```
Comment (OfficialEsco) [inline manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Canary
+PackageVersion: upcoming-411eeac
```
Comment (OfficialEsco) [inline manifests/c/CiderCollective/Cider/Canary/upcoming-411eeac/CiderCollective.Cider.Canary.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by yazninja** [comment]
@OfficialEsco can we keep the version number as is. The development team wants to keep it in line with Github commit hash as this is a nightly build

**Comment by OfficialEsco** [comment]
Well, the commit hash would not be in a upgradable order... anyone installing the package would have to check the repo for the correct commit that they want 🤦‍♂️

Also why are these releases coming from your fork instead of https://github.com/ciderapp/Cider?

**Comment by OfficialEsco** [comment]
> > Well, the commit hash would not be in a upgradable order... anyone installing the package would have to check the repo for the correct commit that they want 🤦‍♂️
> > Also why are these releases coming from your fork instead of https://github.com/ciderapp/Cider?
> 
> well since winget does not support .zip installers

https://github.com/ciderapp/Cider/releases/download/v0.0.1-alpha/Cider.Setup.0.0.1.exe is not zipped ?

**Comment by yazninja** [comment]
> > > Well, the commit hash would not be in a upgradable order... anyone installing the package would have to check the repo for the correct commit that they want 🤦‍♂️
> > > Also why are these releases coming from your fork instead of https://github.com/ciderapp/Cider?
> > 
> > 
> > well since winget does not support .zip installers
> 
> https://github.com/ciderapp/Cider/releases/download/v0.0.1-alpha/Cider.Setup.0.0.1.exe is not zipped ?

this is for the ms store release

**Comment by yazninja** [comment]
> > > Well, the commit hash would not be in a upgradable order... anyone installing the package would have to check the repo for the correct commit that they want 🤦‍♂️
> > > Also why are these releases coming from your fork instead of https://github.com/ciderapp/Cider?
> > 
> > 
> > well since winget does not support .zip installers
> 
> https://github.com/ciderapp/Cider/releases/download/v0.0.1-alpha/Cider.Setup.0.0.1.exe is not zipped ?
> 
> I don't think we can accept your fork as a official source, your alpha is waaaaaaaaaaaay higher than the official source and would upgrade anyone on the official source

how about now, this is canary version and that is why the alpha is way higher, if you compare my fork and og source in upcoming branch, they are the same.

**Pull request closed without merging**